server/middlewares/stats.js

```diff
@@ -41,44 +41,97 @@ const parseTxBodyOutAmount = (txBody) => {
   }
 }
 
-const incrCountersBy = (key, value) => {
-  const [year, month, day] = getSlicedDate()
-  client.incrby(`${key}:total`, value)
-  client.incrby(`${key}:monthly:${year}-${month}`, value)
-  client.incrby(`${key}:daily:${year}-${month}-${day}`, value)
+const initVisitor = (trackingID) => {
+  return ua({
+    tid: trackingID,
+    cid: 555, //google analytics anonymous visitor
+    strictCidFormat: false, //to accept cid integer instead of UUID v4 format
+  })
+}
+
+const trackPageView = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_GA_TRACKING_ID)
+  const args = {
+    dp: argsObj.path,
+    dt: argsObj.hostname,
+    dh: argsObj.title,
+  }
+
+  visitor.pageview(args).send()
+}
+
+const trackEvent = (argsObj) => {
+  const visitor = initVisitor(backendConfig.ADALITE_GA_TRACKING_ID)
+  const args = {
+    ec: argsObj.eventCategory,
+    ea: argsObj.eventAction,
+    el: argsObj.eventLabel,
+    ev: argsObj.eventValue,
+  }
+
+  visitor.event(args).send()
 }
 
 const trackVisits = (req, res, next) => {
+  if (!backendConfig.ADALITE_GA_TRACKING_ID) {
```

Comment:
this was handled here: https://github.com/vacuumlabs/adalite/blob/2a7d7e9ea211beaadd947012f26bd8b652925798/server/index.js#L17 and I would rather keep it like that.